Resources/Prototypes/Entities/Objects/Weapons/Melee/e_sword.yml

```
@@ -312,9 +307,9 @@
         variation: 0.250
     activatedDamage:
         types:
-            Slash: 8
-            Heat: 13
-            Structural: 15
+            Slash: 12
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 10 but found 12

(indentation)